app/src/main/java/com/keylesspalace/tusky/network/MastodonApi.kt

```diff
@@ -253,7 +253,7 @@ interface MastodonApi {
     suspend fun deleteStatus(@Path("id") statusId: String): NetworkResult<DeletedStatus>
 
     @POST("api/v1/statuses/{id}/reblog")
-    suspend fun reblogStatus(@Path("id") statusId: String): NetworkResult<Status>
+    suspend fun reblogStatus(@Path("id") statusId: String, @Query("visibility") visibility: String?): NetworkResult<Status>
```

Comment:
Interestingly this works, but [according to the docs](https://docs.joinmastodon.org/methods/statuses/#form-data-parameters-2), the `visibility` parameter is a form-data paramter and not a query one.

Change the `@Query` to `@Field` and add `@FormUrlEncoded` to the method please.

Reply:
@connyduck fixed and tested